app_radio_module/src/transmitter.c

```diff
@@ -81,6 +90,36 @@ static void lora_tx_task(void *, void *, void *) {
     }
 }
 
+typedef struct {
+    double latitude;
+    double longitude;
+    double altitude;
+} gnss_data_simple;
+
+static void gnss_data_cb(const struct device *dev, const struct gnss_data *data) {
+    if (!ready_to_tx) {
+        return; // timer hasnt expired yet
+    }
+    l_lora_packet_t packet = {0};
+    int port_num = RADIO_MODULE_GNSS_DATA_PORT + RADIO_MODULE_BASE_PORT;
+    packet.port = port_num;
```

Comment:
Can just shorten this

Reply:
resolved https://github.com/RIT-Launch-Initiative/FSW/pull/85/commits/d962f33cf213e381140dcbdbba1db38260e61800